scripts/zones/Rolanberry_Fields_[S]/mobs/Dyinyinga.lua

```diff
@@ -1,11 +1,62 @@
 -----------------------------------
--- Area: Rolanberry Fields [S]
---   NM: Dyinyinga
+--  Area: Rolanberry Fields [S]
+--    NM: Dyinyinga
+-- https://www.bg-wiki.com/ffxi/Dyinyinga
+--  TODO: Gradually builds enfeebling resistance
+-- Dyinyanga tries to stand directly on top of the player and inficts a 0.5 yalm silence amnesia aura.
+-- Has en-slow that overwrites and prevents Haste 2.
+-- When slow is active on the player, the NM moves out to melee at a normal range and stops trying to stand on them until it wears. Aura ceases at this time too.
 -----------------------------------
 local entity = {}
 
 entity.onMobInitialize = function(mob)
-    mob:setMobMod(xi.mobMod.TARGET_DISTANCE_OFFSET, 50)
+    mob:setSpeed(100)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+end
+
+entity.onAdditionalEffect = function(mob, target, damage)
+    return xi.mob.onAddEffect(mob, target, damage, xi.mob.ae.SLOW, { chance = 15, duration = 45, power = 30 })
+end
+
+entity.onMobFight = function(mob, target)
+    if target:getStatusEffect(xi.effect.SLOW) then
+        -- No aura and steps back (if too close)
+        if mob:getMobMod(xi.mobMod.TARGET_DISTANCE_OFFSET) ~= 0 then
+            mob:setMobMod(xi.mobMod.TARGET_DISTANCE_OFFSET, 0)
+            local targetDistance = mob:checkDistance(target)
+            if targetDistance < 2 then
+                -- step back a bit
+                local tarX,       tarZ = target:getXPos(),                target:getZPos()
+                local newX, newY, newZ = mob:getXPos(),    mob:getYPos(), mob:getZPos()
+
+                if targetDistance > .25 then
+                    -- away from target
+                    newX = newX + (newX > tarX and 1 or -1) * math.random(1, 3)
+                    newZ = newZ + (newZ > tarZ and 1 or -1) * math.random(1, 3)
+                else
+                    -- random direction
+                    newX = newX + (math.random(1, 2) == 1 and 1 or -1) * math.random(1, 3)
+                    newZ = newZ + (math.random(1, 2) == 1 and 1 or -1) * math.random(1, 3)
+                end
+
+                mob:pathTo(newX, newY, newZ)
+            end
+        end
+    else
+        -- Aura and closes the gap
+        mob:setMobMod(xi.mobMod.TARGET_DISTANCE_OFFSET, 50)
+
+        for _, entry in pairs(mob:getEnmityList()) do
```

Comment:
Is there any reason the real aura effect was not used?
Probably something like
```
mob:setMod(xi.mod.AURA_SIZE, 320) -- 6.25 * 320 / 100 = 20'
mob:addStatusEffectEx(xi.effect.COLURE_ACTIVE, xi.effect.COLURE_ACTIVE, 6, 3, 0, xi.effect.SLOW, 50, xi.auraTarget.ENEMIES, xi.effectFlag.AURA)
```
this is assuming that the aura doesn't have a visible bubble, not sure how to do that offhand, you can see if it uses a real aura in a capture if the status effect for SLOW has its duration constantly update.

Reply:
There is no visible bubble. 

I had no idea we had actual aura code 

I'll check the capture for that, too

Reply:
Well captures confirm it's an aura.

I tried what you suggested and got only the second addEffect to stick:

![image](https://github.com/LandSandBoat/server/assets/131182600/7b3d51a3-3d41-478e-8367-92061c2b3eb1)

Reply:
currently you can only have one COLURE_ACTIVE effect, auras were designed with GEO in mind (and I dont think anyone was aware a mob could have _two_)... hmmm

Reply:
auras and geo spells are slightly different while performing similar tasks. A GEO's spell can normally only applies one to one target: one on itself, one on a loupan, one in a 2nd party member via use of a JA...


While a mobs aura and throw a ton of crap at you at SE's whim

Reply:
Set one to `mob:addStatusEffectEx(xi.effect.NONE xi.effect.NONE,`
?

We can't see them anyway and you can mark that effect as undispelable, etc

Reply:
How would it know to be a colour effect of we set it to none

Reply:
Sorry for questioning, it works!

I guess the aura effect stuff is just global for all effects, neat